.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,239 @@
+        env:
+          MOCHA_REMOTE_CONTEXT: missingServer
+          MOCHA_REMOTE_REPORTER: mocha-github-actions-reporter
+  react-native-ios:
+    name: React Native on iOS (${{ matrix.type }})
+    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        type: [Release, Debug]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+      # Setup tools
+      - name: Install tools
+        run: brew install watchman
+      # TODO: Expose a way to skip building the Node.js native module
+      - name: Patch package.json to avoid building Node.js native module
+        run: cat <<< $(jq '.scripts.install = "echo skipping building Node.js binary"' < package.json) > package.json
+
+      # Caches
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: npm-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            npm-
+      - uses: actions/cache@v2
+        name: Restore iOS build folder from cache
+        with:
+          path: react-native/ios/build
+          key: ios-build-dir-${{ hashFiles('src/**', 'react-native/ios/**', 'vendor/**') }}
+          restore-keys: |
+            ios-build-dir-
+      - uses: actions/cache@v2
+        name: Restore xcframework from cache
+        id: cache-xcframework
+        with:
+          path: react-native/ios/realm-js-ios.xcframework
+          key: xcframework-${{ hashFiles('src/**', 'react-native/ios/**', 'vendor/**') }}
+      - uses: actions/cache@v2
+        name: Restore Xcode "derived data" from cache
+        with:
+          path: ~/Library/Developer/Xcode/DerivedData
+          key: xcode-derived-data
+      - uses: actions/cache@v2
+        name: Restore CocoaPods from cache
+        with:
+          path: integration-tests/environments/react-native/ios/Pods
+          key: cocoapods-${{ hashFiles('integration-tests/environments/react-native/ios/Podfile.lock') }}
+          restore-keys: | # Enables running from an old build
+            cocoapods-
+
+      - run: npm ci
+      # Build the iOS xcframework (if cache missed)
+      - run: ./scripts/build-ios.sh -c ${{ matrix.type }} -s
+        if: ${{ steps.cache-xcframework.outputs.cache-hit != 'true' }}
+      # Bootstrap lerna sub-packages (builds the packages, the Realm JS native module and pod install)
+      - run: npx lerna bootstrap --scope '{realm-integration-tests,realm-react-native-tests}' --include-dependencies
+      # Run the tests
+      - name: Run tests (iOS / Native)
+        run: npm run test:ios --prefix integration-tests/environments/react-native
+        env:
+          MOCHA_REMOTE_CONTEXT: missingServer
+          MOCHA_REMOTE_REPORTER: mocha-github-actions-reporter
+          MOCHA_REMOTE_EXIT_ON_ERROR: true
+      - name: Run tests (iOS / Chrome Debugging)
```

Comment:
Disable running these tests and create a ticket to implement the fixes and reenable the tests.

Reply:
I created https://jira.mongodb.org/browse/RJS-1166 to track this.